airflow-core/src/airflow/ui/src/i18n/locales/de/components.json

```
@@ -0,0 +1,119 @@
+{
+  "backfill": {
+    "affectedMultiple": "{{count}} Läufe werden ausgelöst.",
+    "affectedNone": "Keine Läufe entsprechend en Kriterien.",
+    "affectedOne": "1 Lauf wird ausgelöst.",
+    "backwards": "Verarbeitung in Rückwärtiger Reihenfolge",
```

```suggestion
    "backwards": "Verarbeitung in rückwärtiger Reihenfolge",
```

Unsure about what makes more sense here, feel free to ignore :)

---

No, sorry, you are correct.